kairon/shared/channels/whatsapp/bsp/dialog360.py

```
@@ -376,4 +376,39 @@ def delete_media_file(media_id: str, channel_config):
-        return "Media file deleted successfully"
\ No newline at end of file
+        return "Media file deleted successfully"
+
+    @staticmethod
+    async def get_media_content(bot: str, bsp_type: str, media_id: str):
+        """
+        Returns the PDF bytes from media ID.
+        """
+        connector_type = "whatsapp"
+        try:
+            media_doc = UserMediaData.objects.get(media_id=media_id)
```

Ruff (0.15.1)

[error] Local variable `media_doc` is assigned to but never used

Remove assignment to unused variable `media_doc`

(F841)